### ISSUE TITLE
[ENHANCEMENT] Allow sessions longer than 1 hour for happier operators

### ISSUE THREAD

**Issue by srabraham** [burningmantech/ranger-ims-server#1299]
I did a few operator shifts this year, and I noted every fellow operator complaining about getting logged out of IMS many times during their shifts. This can be majorly inconvenient when it happens during busy times.

The state of things now is that an IMS session will always time out within 1 hour of its start [1]. That value is set [here in our code](https://github.com/burningmantech/ranger-ims-server/blob/1792c93c4c10408beaca2780bc496f562b9300e2/src/ims/run/_command.py#L59).

The easiest thing would be to bump that value up from 1 hour to 7 hours. That would cover any normal operator shift. I'm open to alternative suggestions. I just don't think the status quo is reasonable for operators (nor really for shift leads).

FYI @justfred @wsanchez @chaoticbear, please provide any input.

<details><summary>[1] Gratuitous details on sessionTimeout</summary>
The Twisted documentation says that the sessionTimeout "attribute gives the number of seconds a session may go without being accessed before it expires". That led me to believe that the session should be getting refreshed regularly when the user clicks around within IMS. However, that doesn't happen, because nowhere are we invoking Twisted's "session.touch()" or "session.getSession()", which we'd need to do to actually refresh the session.

That suggests an alternative solution, in which we start calling session.touch() sometimes. There's another problem with that idea though...due to all the automated calls that happen on event changes, we'd constantly be calling session.touch() on all sessions during event week, and a session would likely never time out.
</details>

**Comment by sdebaun** [comment]
logoutcomms, dearest --

I operatored for my first time this past year, did a couple shifts and loved it.  I heard the same feedback from other operators.  I ran into it, and while it didn't bug me that much, it's clearly friction to other ops.

Stepping back to look at the intent of a timeout here, to me it seems like: there's a risk of misattributing access if operator A leaves themselves logged in after their shift, then the next operator B might just start using the app.  Then all of op B's use is recorded as being done under op A.  Operational (har har) confusion imminent!

Just arbitrarily forcing op's to log back in every hour really seems like the "f you" solution.

What are some better solutions?  Solutions that still mitigate the risk of operational confusion, but also don't introduce unneeded user friction?  Solutions that don't take a lot of effort to implement?

Couple I've thought of while chewing on this a bit:

a) Have the logged-in-user display in the UI start to pulse around every shift change.  A single popup happens at the scheduled shift change to ask "Are you still operator A?  No | Ask again in [dropdown] hours."  Get fancy have some workflow for if they say "No", then ask when they took the chair, log them in, and update all the user activity since then to the new user.
Pros: much better balance between reducing risk to incorrectly attributing usage, and annoying the user. 
Cons: "Dearest?"  "Yes?"  "Isn't that a lot of lift?" "Yes..." "Be quiet Dearest"

b) Automatically log them out when the shift change happens.
Cons: Still some risk of incorrectly attributing usage, eg for when an op user switches out halfway through shift for whatever reason.  Also still will annoy ops that work a double, or work past shift end -- but will only annoy them ONCE.
Pros: "Dearest?"  "Yes?"  "Much better."
